src/descriptors/mod.rs

```diff
@@ -17,7 +17,7 @@ pub use keys::*;
 pub mod analysis;
 pub use analysis::*;
 
-const WITNESS_FACTOR: usize = 4;
+pub const WITNESS_FACTOR: usize = 4;
```

Comment:
In the future we should avoid rolling our own constant and instead use rust-bitcoin's.